backend/src/kitconcept/intranet/services/feedback_contact_form.py

```diff
@@ -0,0 +1,293 @@
+    return value
+
+
+class FeedbackPostContactForm(Service):
+    """Submit the contact form and send an email depending of the category."""
+
+    def reply(self):
+        """Send an email to the responsible person and the person who submitted the feedback."""
+        parent_object = self.context
+        cc_email = get_registry_value(
+            IIntranetSettings, "feedback_cc_email", "kitconcept.intranet"
+        )
+        default_email = get_registry_value(
+            IIntranetSettings, "default_feedback_email", "kitconcept.intranet"
+        )
+        data = json_body(self.request)
+        expander = getMultiAdapter(
+            (parent_object, self.request),
+            IExpandableElement,
+            name="lcm",
+        )
+        expanded_data = expander(expand=True)
+        responsible_person_uuid = (
+            expanded_data.get("lcm", {})
+            .get("responsible_person", {})
+            .get("value", "no-person")
+        )
+        feedback_person_uuid = parent_object.feedback_person
+        feedback_member = (
+            uuidToObject(feedback_person_uuid) if feedback_person_uuid else None
+        )
+        responsible_member = (
+            uuidToObject(responsible_person_uuid) if responsible_person_uuid else None
+        )
+        feedback_recipient_email = (
+            getattr(feedback_member, "contact_email", None)
+            or getattr(responsible_member, "contact_email", None)
+            or ""
+        )
+        if not feedback_recipient_email:
+            feedback_recipient_email = default_email
+        data["feedback_recipient_email"] = feedback_recipient_email
+        data["cc_email"] = cc_email
+        validated = self._validate(data, parent_object)
+        self._send_email(validated)
+        self._send_confirmation_email(validated)
+        return self.reply_no_content()
+
+    def _translate(self, msg):
+        return translate(msg, context=self.request)
+
+    def _validate(self, data, parent_object):
+        allowed_emails_domain = get_registry_value(
+            IIntranetSettings, "allowed_email_domains", "kitconcept.intranet"
+        )
+        reporter_email = data.get("email")
+        name = data.get("name")
+        feedback = data.get("feedback")
+        feedback_recipient_email = data.get("feedback_recipient_email")
+
+        if not feedback_recipient_email:
+            raise BadRequest(
+                self._translate(
+                    _(
+                        "No responsible person or default feedback email is configured. "
+                        "Please contact the site administrator."
+                    )
+                )
+            )
+
+        if not feedback:
+            raise BadRequest(self._translate(_("Please enter your feedback.")))
+
+        if not reporter_email or (
+            allowed_emails_domain
+            and not any(
+                reporter_email.endswith(domain) for domain in allowed_emails_domain
```

Comment:
```suggestion
                reporter_email.endswith("@" + domain) for domain in allowed_emails_domain
```

Otherwise someone can submit myemail@hackerkitconcept.com when it's supposed to only allow kitconcept.com

Reply:
I thought the editor would save the domain for something like `@kitconcept.com`.

Reply:
done